src/index.js

```diff
@@ -184,8 +185,11 @@ class CreditCardInput extends Component<Props, State> {
   };
 
   handleCardExpiryBlur = (e: SyntheticInputEvent<*>) => {
-    if (!payment.fns.validateCardExpiry(e.target.value)) {
-      this.setFieldInvalid('Expiry date is invalid');
+    let cardExpiryBlur = e.target.value.split(' / ').join('/');
```

Comment:
`let` to `const`, and can we perhaps name this just `cardExpiry`?

Reply:
Oh, yes, for some reason, my limited js knowledge was pushing me to believe that `const cardExpiry` might overlap with the corresponding one in `handleCardExpiryChange`, but of course, `const`is block-scoped, everything should be fine.

Fixed